scripts/internal/rollout_price.go

```diff
@@ -79,7 +79,7 @@ func SyncPlanPrices() error {
 	planPrices := make([]*price.Price, 0)
 	meterMap := make(map[string]*meter.Meter)
 	for _, price := range prices {
-		if price.PlanID == planID && price.TenantID == tenantID && price.EnvironmentID == environmentID {
+		if price.EntityID == planID && price.TenantID == tenantID && price.EnvironmentID == environmentID && price.EntityType == types.PRICE_ENTITY_TYPE_PLAN {
```

Comment:
Will also need handling to roll out new pricing additions in the addons. Can do this separately as well after this PR is live